### PR TITLE
Permission set TPL Standard User Updated

### PR DESCRIPTION
**Pull request by mhussaincgi**

### CODE REVIEW

**Comment by cgijeffolsen** [comment]
Before deploying please revisit Health Care Costs as Record Type Name was not available to the document generation process when using a TPL standard user. This field is needed for identification or records. @mhussaincgi @NataliaNikishina

File: TPL/force-app/main/default/permissionsets/TPL_Standard_User.permissionset-meta.xml [hunk 1]
```diff
@@ -467,6 +467,11 @@
         <editable>true</editable>
         <field>Healthcare_Cost__c.Description__c</field>
         <readable>true</readable>
+    </fieldPermissions>
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/permissionsets/TPL_Standard_User.permissionset-meta.xml:470]: Access to fields added to supporting writing of new records and filtering needed for document generation.

**Review by cgijeffolsen — APPROVED** [review]
@mhussaincgi @NataliaNikishina Approved

**Review by NataliaNikishina — APPROVED** [review]
@mhussaincgi approved

**Pull request merged**